package.json

```diff
@@ -66,7 +66,7 @@
     "gen-types": "npx -p typescript tsc"
   },
   "engines": {
-    "node": ">=20"
+    "node": ">=23"
```

Comment:
Why? We have users on 20, so let's try and support a broad range of users.

Reply:
Why?  Was trying to keep the nvmrc, Dockerfile and engine versions in-sync.  At the moment there are 3 different versions being used...